src/utils/network/core.cljs

```diff
@@ -41,7 +41,12 @@
        (map :error)
        (not-any? identity)))
 
-(defn chain-id-available?
-  [current-networks network]
-  (let [chain-id (get-in network [:config :NetworkId])]
-    (every? #(not= chain-id (get-in % [1 :config :NetworkId])) current-networks)))
```

Comment:
NOT RELATED: Removed since this namespace is not related to wallet stuff, and also it has 0 usages